src/modules/organizations/containers/media-container.jsx

```diff
@@ -0,0 +1,231 @@
+/* eslint-disable no-console */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import apiClient from 'panoptes-client/lib/api-client';
+
+import MediaArea from '../../common/components/media-area';
+import { setCurrentOrganization } from '../action-creators';
+import { organizationShape } from '../model';
+
+export class MediaContainer extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
not entirely sure what should go to redux store; nothing, just `media`, all things (`media, pendingFiles, pendingMedia`) or other?

Reply:
I think the generic methods that could apply to either an organization or a project should be in its own container component and set to the local state. This isn't really state that the whole app needs to be aware of.